config/services.yaml

```diff
@@ -49,3 +49,18 @@ services:
     App\Security\Authenticator\GitHubAuthenticator:
         arguments:
             $allowedGitHubOrganization: '%env(APP_ALLOWED_GITHUB_ORGANIZATION)%'
+
+    client:
```

Comment:
this name is too generic / vague. Please use something like `http.guzzle.client`

Reply:
Nevermind: use the classname for the ie, same after